docs/wp-scripts.md

```diff
@@ -3,15 +3,16 @@
 ## Lint
 
 This workflow runs [ESLint](https://eslint.org/), [Stylelint](https://stylelint.io/),
-and [markdownlint](https://github.com/DavidAnson/markdownlint) wrapped in the
-the [`@wordpress/scripts`](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-scripts/)
+and [markdownlint](https://github.com/DavidAnson/markdownlint) wrapped in the the [
+`@wordpress/scripts`](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-scripts/)
```

Comment:
```suggestion
and [markdownlint](https://github.com/DavidAnson/markdownlint) wrapped in the
[`@wordpress/scripts`](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-scripts/)
```

Reply:
Thanks for the suggestion, which I agree. However, PhpStorm automatically applied all changes, and I'd rather stick to that if that's okay. There are other occurrences of weird line separations, like here: https://github.com/inpsyde/reusable-workflows/pull/156/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R9-R10 and here: https://github.com/inpsyde/reusable-workflows/pull/156/files#diff-c55cc7d1f7f61335e49de6bc171223be1afa8985e9d9a81b536081bd9a65fd8dR34-R35.

Reply:
Sure. Then maybe we can just remove the 2x "the".

Reply:
Sorry, I totally overlooked that. 🤦🏽‍♂️